worker/caasapplicationprovisioner/application.go

```diff
@@ -98,6 +98,28 @@ func (a *appWorker) Wait() error {
 }
 
 func (a *appWorker) loop() error {
+	// TODO(sidecar): support more than statefulset
+	app := a.broker.Application(a.name, caas.DeploymentStateful)
```

Comment:
curious, why move this?

Reply:
This is because `app` is used for the call to `a.dead` in the new block, so I needed to move it up.